### DISCUSSION TITLE
打开 MFSU 后启动报错

### DISCUSSION THREAD

**Discussion by xiaohp** [umijs/umi#7491] [Q&A]
报错信息
![image](https://user-images.githubusercontent.com/23056320/138231863-34f50c9b-c5ff-43a7-a2cb-9308fc7a9e6d.png)

文字版：
Unhandled Rejection (ChunkLoadError): Loading chunk mf-dep_vendors failed.
(missing: http://localhost:8000/h5-deal/vendors.526babb3.async.js)
while loading "./$CWD$/node_modules/umi/node_modules/@umijs/runtime" from webpack/container/reference/mf
while loading "./$CWD$/node_modules/@umijs/babel-preset-umi/node_modules/@babel/runtime/regenerator" from webpack/container/reference/mf
while loading "./react" from webpack/container/reference/mf
while loading "./$CWD$/node_modules/react-helmet" from webpack/container/reference/mf
while loading "./regenerator-runtime/runtime" from webpack/container/reference/mf
▼ 8 stack frames were expanded.
__webpack_require__.f.j
http://localhost:8000/h5-deal/mf-va_remoteEntry.js:273:29
(anonymous function)
http://localhost:8000/h5-deal/mf-va_remoteEntry.js:67:40
Function.__webpack_require__.e
http://localhost:8000/h5-deal/mf-va_remoteEntry.js:66:67
./$CWD$/node_modules/umi/node_modules/@umijs/runtime
http://localhost:8000/h5-deal/mf-va_remoteEntry.js:348:43
get
http://localhost:8000/h5-deal/mf-va_remoteEntry.js:508:23
handleFunction
webpack:/h5-deal/webpack/runtime/remotes loading:449
onInitialized
webpack:/h5-deal/webpack/runtime/remotes loading:461
(anonymous function)
webpack:/h5-deal/webpack/runtime/remotes loading:451
▲ 8 stack frames were expanded.

启动时没有 warning，从报错信息没找到原因，关闭 mfsu 后正常

umi 版本：3.5.20
node 版本：v14.17.5
操作系统版本：macOS 11.6

**Comment by gaven-tuan** [comment]
解决了吗，请问是什么原因？
